### PR TITLE
ensure a few "percentX" property doesn't return a boolean

### PR DESCRIPTION
**Pull request by mrbuds**
fix type error when they are use in conditions

fixes an error reported on discord https://pastebin.com/j8CHBhpi using this aura https://wago.io/8KnH9fmY3

### CODE REVIEW

**Comment by mrbuds** [comment]
i'm not sure what is best value to return if total == 0 between 0 or 100, i made the choice to use 100, it may not be correct

File: WeakAuras/Prototypes.lua [hunk 1]
```diff
@@ -2061,7 +2061,7 @@ Private.event_prototypes = {
         name = "percentXP",
         display = L["Experience (%)"],
         type = "number",
-        init = "total ~= 0 and (value / total) * 100",
+        init = "total ~= 0 and (value / total) * 100 or 100",
```
Comment (InfusOnWoW) [inline WeakAuras/Prototypes.lua:2064]: "or nil" should work.

Comment (mrbuds) [reply WeakAuras/Prototypes.lua]: that sound more sane, it's updated

**Comment by InfusOnWoW** [comment]
Well now the commit message is wrong...

File: WeakAuras/Prototypes.lua (hunk 1 above)
Comment (mrbuds) [reply WeakAuras/Prototypes.lua]: > 
> 
> Well now the commit message is wrong...

indeed, fixed

**Pull request merged**